src/protocol/sort/apply_sort/mod.rs

```diff
@@ -91,7 +92,7 @@ mod tests {
         });
         let expected = permutation.apply_slice(&sidecar);
 
-        let result = world
+        let result: [Vec<AttributionInputRow<Fp32BitPrime>>; 3] = world
```

Comment:
Do we need to explicitly give it a type?

Reply:
Yes.